js/ui/components/clubhouse.js

```diff
@@ -650,8 +668,11 @@ var ClubhouseButtonManager = new Lang.Class({
 
         Main.layoutManager.addChrome(this._openButton);
 
-        this._closeButton =
-            new St.Button({ child: new St.Icon({ style_class: 'clubhouse-close-button-icon' }) });
+        let icon = new St.Icon({ style_class: 'clubhouse-close-button-icon',
+                                 width: CLUBHOUSE_BUTTON_SIZE,
+                                 height: CLUBHOUSE_BUTTON_SIZE
```

Comment:
This is so that the close button is placed exactly at the same position as the open icon.

Reply:
Ditto